website/docs/patterns/configuring_plugins.md

```diff
@@ -39,7 +39,7 @@ class Falcon9Conf:
 
 ```python
 @dataclass
-class Simulation:
+class Sim:
```

Comment:
We can keep the class name `Simulation`, no one said that the config group name must match the config class name exactly.